src/main/cli/cli.c

```diff
@@ -1276,74 +1282,109 @@ static void cliSerialPassthrough(char *cmdline)
                 pinioDtr = atoi(tok);
 #endif /* USE_PINIO */
             }
-
+            break;
+        case 4:
+            leftSidePortID = atoi(tok);
+            break;
+        case 5:
+            leftSideBaud = atoi(tok);
+            break;
+        case 6:
+            if (strcasestr(tok, "rx")) {
```

Comment:
This could be extracted as a function and shared with the parsing already done above.

Reply:
ok